test/helpers.py

```diff
@@ -119,7 +119,7 @@ def build_goaway_frame(self,
         f = GoAwayFrame(0)
         f.error_code = error_code
         f.last_stream_id = last_stream_id
-        f.additional_data = additional_data
+        f.additional_data = additional_data or b''
```

Comment:
I wanted to test all three cases: https://github.com/python-hyper/hyper-h2/pull/231/files#diff-19d167383ddbd75543abdbc92e6b0698R853
Or is that also not necessary?